pcsx2/gui/MainFrame.cpp

```diff
@@ -322,6 +338,147 @@ static int GetPluginMenuId_Name( PluginsEnum_t pid )
+{
+	m_menuConfig.Append(MenuId_Config_SysSettings,	_("Emulation &Settings...") );
+	m_menuConfig.Append(MenuId_Config_McdSettings,	_("&Memory cards...") );
+	m_menuConfig.Append(MenuId_Config_BIOS,			_("&Plugin/BIOS Selector...") );
+
+	m_menuConfig.AppendSeparator();
+
+	m_menuConfig.Append(MenuId_Config_GS,		_("&Video (GS)"),		m_PluginMenuPacks[PluginId_GS]);
+	m_menuConfig.Append(MenuId_Config_SPU2,		_("&Audio (SPU2)"),		m_PluginMenuPacks[PluginId_SPU2]);
+	m_menuConfig.Append(MenuId_Config_PAD,		_("&Controllers (PAD)"),m_PluginMenuPacks[PluginId_PAD]);
+	m_menuConfig.Append(MenuId_Config_DEV9,		_("&Dev9"),				m_PluginMenuPacks[PluginId_DEV9]);
+	m_menuConfig.Append(MenuId_Config_USB,		_("&USB"),				m_PluginMenuPacks[PluginId_USB]);
+	m_menuConfig.Append(MenuId_Config_FireWire,	_("&Firewire"),			m_PluginMenuPacks[PluginId_FW]);
+
+	m_menuConfig.AppendSeparator();
+	m_menuConfig.Append(MenuId_Config_Multitap0Toggle,	_("Multitap &1"),	wxEmptyString, wxITEM_CHECK );
+	m_menuConfig.Append(MenuId_Config_Multitap1Toggle,	_("Multitap &2"),	wxEmptyString, wxITEM_CHECK );
+
+	m_menuConfig.AppendSeparator();
+
+	m_menuConfig.Append(MenuId_ChangeLang,			L"Change &language..." ); // Always in English
+	m_menuConfig.Append(MenuId_Config_ResetAll,	_("C&lear all settings..."),
+		AddAppName(_("Clears all %s settings and re-runs the startup wizard.")));
+}
+
+void MainEmuFrame::CreateWindowsMenu()
+{
+	m_menuWindow.Append(MenuId_Debug_Open,		_("&Show Debug"),	wxEmptyString, wxITEM_CHECK );
+
+	m_menuWindow.Append( &m_MenuItem_Console );
+#if defined(__unix__)
+	m_menuWindow.AppendSeparator();
+	m_menuWindow.Append( &m_MenuItem_Console_Stdio );
+#endif
+}
+
+void MainEmuFrame::CreateCaptureMenu()
+{
+	m_menuCapture.Append(MenuId_Capture_Video, _("Video"), &m_submenuVideoCapture);
+	m_submenuVideoCapture.Append(MenuId_Capture_Video_Record, _("Start Recording"));
+	m_submenuVideoCapture.Append(MenuId_Capture_Video_Stop, _("Stop Recording"))->Enable(false);
+
+	m_menuCapture.Append(MenuId_Capture_Screenshot, _("Screenshot"));
+}
+
+void MainEmuFrame::CreateRecordMenu()
+{
+	m_menuRecording.Append(MenuId_Recording_New, _("New"));
+	m_menuRecording.Append(MenuId_Recording_Stop, _("Stop"))->Enable(false);
+	m_menuRecording.Append(MenuId_Recording_Play, _("Play"));
+	m_menuRecording.AppendSeparator();
+	m_menuRecording.Append(MenuId_Recording_VirtualPad_Port0, _("Virtual Pad (Port 1)"));
+	m_menuRecording.Append(MenuId_Recording_VirtualPad_Port1, _("Virtual Pad (Port 2)"));
+}
+
+void MainEmuFrame::CreateHelpMenu()
+{
+	m_menuHelp.Append(MenuId_Help_GetStarted,				_("&Getting Started") );
+	m_menuHelp.Append(MenuId_Help_Compatibility,			_("&Compatibility") );
+	m_menuHelp.AppendSeparator();
+	m_menuHelp.Append(MenuId_Help_Forums,					_("&Pcsx2 Forums") );
+	m_menuHelp.Append(MenuId_Help_Website,					_("&Pcsx2 Website") );
+	m_menuHelp.Append(MenuId_Help_Wiki,						_("&Pcsx2 Wiki") );
+	m_menuHelp.Append(MenuId_Help_Github,					_("&Pcsx2 Github Project") );
```

Comment:
Dunno about having them duplicated both here and the about page. Also you can drop the pcsx2 part on the menus.

Reply:
I'm not sure what I'd replace them with in the About window, though, and the License link isn't in the menu, currently. The About is already pretty bare. I figured I'd just leave it as is for now.

I went ahead and dropped "Pcsx2" from the names, and rearranged them slightly.